test/unit/mongo_logger.test.ts

```diff
@@ -0,0 +1,52 @@
+import { expect } from 'chai';
+
+import {
+  MongoLoggableComponent,
+  MongoLogger,
+  MongoLoggerOptions,
+  SeverityLevel
+} from '../../src/mongo_logger';
+
+describe('Logger', function () {
+  describe('options parsing', function () {
+    let loggerOptions: MongoLoggerOptions;
```

Comment:
good point. i'm gonna hold on creating tests for that logDestination logic in case it gets refactored as a result of our conversation [here](https://github.com/mongodb/node-mongodb-native/pull/3475#discussion_r1030746593). leaving this conversation unresolved as a reminder to ultimately write these tests